scripts/init.sh

```diff
@@ -31,6 +31,26 @@ cap db create
 # Create default user:
 cap users create info@inveniosoftware.org -a --password infoinfo
 
+cap users create cms@cap.ch -a --password cmscms
```

Comment:
Works for me locally 👍 but we'd better not use `cap.ch` as an example domain name... I'll fix it locally before merging this PR in order to save time.

We can muse later that this could be done for venv style of developments, not only for docker.  (I have `scripts/*` refactoring in mind we can talk about after the workshop.)